wptdash/date.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from datetime import date
+import math
+
+
+def get_quarter_start_date(quarter, year):
+    quarter_first_month = quarter * 3 - 2
+    quarter_start = date(year, quarter_first_month, 1)
+    return quarter_start
+
+
+def get_default_start_end():
+    month_end_days = {
+        3: 31,
+        6: 30,
+        9: 30,
+        12: 31
+    }
+    today = date.today()
+    month = today.month
+    year = today.year
+    quarter = math.ceil(month / 3)
```

Comment:
Is this python 2 or 3? Integer division is the default in 2, so the quarters will be `[0.0, 0.0, 1.0, 1.0, 1.0, 2.0, 2.0, 2.0, 3.0, 3.0, 3.0, 4.0]`.

`(month + 2) // 3` does the right thing in both python 2/3 it seems.

Reply:
`month / 3.0` looks better to me.

Reply:
This is Python3, but I'll use the float.